src/components/listings/AddListing.js

```diff
@@ -0,0 +1,92 @@
+// npm packages
+import React from "react";
+import DayPicker from "react-day-picker";
+
+// styles
+import "./index.css";
+import "react-day-picker/lib/style.css";
+
+const AddListing = () => {
+  const handleInputChange = (event) => {
+    const target = event.target;
+    const name = target.name;
+    // const value = target.value;
+
+    switch (name) {
```

Comment:
Why are the cases empty?

Reply:
I building the cases logic, wait for the next pull request.